### PR TITLE
LPS-90257 Add dropdown menu to the User Personal Bar

### PR DESCRIPTION
**Pull request by pei-jung**
This adds the dropdown menu to the user avatar when it's clicked. Each menu item still behaves the old way, showing the portlet in the admin/control panel theme. 

@jorgeferrer are these changes look good to you?

/cc @stsquared99 @alee8888

### CODE REVIEW

**Comment by pei-jung** [comment]
**Screenshots of the changes:**

screen shot 2019-02-21 at 6 46 55 pm
screen shot 2019-02-21 at 6 47 12 pm

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by JorgeFerrer** [comment]
Hey Pei-Jung,

Good job! Judging by the screenshots, they do look good to me.
I'll ask Dennis to check the possibility of renaming the OAuth2 entry so that it's shorter and less technical

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/SignOutUserPersonalMenuEntry.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"product.navigation.user.personal.menu.entry.order:Integer=100",
+		"product.navigation.user.personal.menu.group:Integer=400"
+	},
+	service = UserPersonalMenuEntry.class
+)
+public class SignOutUserPersonalMenuEntry extends BaseUserPersonalMenuEntry {
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/SignOutUserPersonalMenuEntry.java]: If we remove the implementation-related methods from the interface (the ones that are not used by the caller of this API), then this class can just implement the interface directly without extending from the base class.

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/MyProfileUserPersonalMenuEntry.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"product.navigation.user.personal.menu.entry.order:Integer=100",
+		"product.navigation.user.personal.menu.group:Integer=100"
+	},
+	service = UserPersonalMenuEntry.class
+)
+public class MyProfileUserPersonalMenuEntry extends BaseUserPersonalMenuEntry {
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/MyProfileUserPersonalMenuEntry.java]: Same as my comment on `SignOutUserPersonalMenuEntry`

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/MyDashboardUserPersonalMenuEntry.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ */
+@Component(
+	immediate = true,
+	property = {
+		"product.navigation.user.personal.menu.entry.order:Integer=200",
+		"product.navigation.user.personal.menu.group:Integer=100"
+	},
+	service = UserPersonalMenuEntry.class
+)
+public class MyDashboardUserPersonalMenuEntry
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/product/navigation/user/personal/menu/MyDashboardUserPersonalMenuEntry.java]: Same as my comment on `SignOutUserPersonalMenuEntry`

**Comment by pei-jung** [comment]
Thanks @JorgeFerrer @drewbrokke!

PR resent as https://github.com/drewbrokke/liferay-portal/pull/760

**Pull request closed without merging**